books/RayTracingInOneWeekend.html

```diff
@@ -1429,8 +1430,56 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     [Listing [rtweekend-initial]: <kbd>[rtweekend.h]</kbd> The rtweekend.h common header]
 
+All main program files will include `rtweekend.h` first, so most other header files (where the bulk
+of our code will reside) can assume that these definitions are already available. (Headers included
+inside `rtweekend.h` still need to include any of their dependencies.) We'll make some updates with
+this assumption in mind.
+
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ delete
+    #include "vec3.h"
+
+    #include <iostream>
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    [Listing [assume-rtw-color]: <kbd>[color.h]</kbd> Assume rtweekend.h in color.h]
```

Comment:
Don't love the term "Assume".  

`Cleanup x.h in favor of rtweekend.h` might read a little better.

Reply:
"in favor of" doesn't make sense to me. I can change to "Assume rtweekend.h inclusion".